client/src/app/group-details/group-details.component.html

```diff
@@ -1,36 +1,37 @@
-<h2>{{ group.name }} group</h2>

```

Comment:
Please change your Git settings, not to substitute CR with CRLF.

Reply:
Our CTO is puzzled.